src/main/webapp/i18n/de/level.json

```
@@ -19,7 +19,7 @@
             "name": "Name",
             "description": "Description",
             "requiredScore": "Required Score",
-            "instantMultiplier": "Skill Score Multiplier",
+            "instantMultiplier": "Skill Score Multiplikator",
```

This should have been fixed together with the other i18n fixes.
```suggestion
            "instantMultiplier": "Skill Score Multiplier",
```